sema4ai/vscode-client/src/extension.ts

```diff
@@ -705,6 +707,28 @@ export async function doActivate(context: ExtensionContext, C: CommandRegistry)
         }
     });
 
+    C.register(SEMA4AI_PACKAGE_PUBLISH_TO_DESKTOP_APP, async () => {
+        const selected = await listAndAskRobotSelection(
+            "Please select the Task/Action Package for which you'd like to rebuild the environment",
```

Comment:
Should the text copy here just talk about Action Package and publish to Sema4.ai Desktop?